core/Config.php

```diff
@@ -514,22 +514,23 @@ protected function getUncached($class, $name, $sourceOptions, &$result, $suppres
 			if ($extraSources) $sources = array_merge($sources, $extraSources);
 		}
 
-		$value = $nothing = null;
```

Comment:
Another observation: `$value` needs to be reset to `null` inside the `foreach($sources...` loop: it *was* being set back to `null` by `$value = $statics->get($staticSource, $name, $nothing);`, but that has been removed.